dclab/definitions/meta_parse.py

```diff
@@ -62,8 +84,10 @@ def lcstr(astr):
 
 #: maps functions to their expected output types
 func_types = {
```

Comment:
By the way @paulmueller I didn't realise I had to add these mappings. I came across a rather ambiguous error while testing my own code. Something along the line of `TypeError: isinstance() arg 2 must be a type, a tuple of types... etc`. It took me a while to find the culprit (it's some isinstance check in `dclab_verify_dataset`.

So I added some parts to the dfn tests that check that there is a correct mapping. To be future-proof, should I add a test that gathers all the functions in this module and ensures there is a mapping for it?